app/controllers/ability.rb

```diff
@@ -0,0 +1,52 @@
+class Ability
```

Comment:
Just a curious question
Why are we moving abilities from model to a controller?

Reply:
Because we need an ability file _per api version_

Models are not versioned, so it wouldn't work